src/material/expansion/expansion.spec.ts

```diff
@@ -487,6 +496,15 @@ describe('MatExpansionPanel', () => {
         expect(header.classList).not.toContain('mat-expanded');
       });
 
+    it('should toggle the tabindex of the header', () => {
```

Comment:
Maybe we can be a bit more explicit here (something around)?

```suggestion
    it('should update the tabindex if the header becomes disabled', () => {
```